config/locales/en.yml

```diff
@@ -969,22 +971,23 @@ en:
           amount: *net_amount
           remove: *remove
           rate: *rate
+          rate_html: Rate <span class="govuk-visually-hidden">in pounds for %{context}</span>
           help_how_we_calculate_rate_title: *how_is_this_calculated_help
           help_how_we_calculate_rate_body: We calculate the rate based upon the fee scheme, case type, offence, advocate category and fee type.
           daf:
-            quantity_hint: *empty_input
+            quantity_hint: Number of daily attendance fee (3 to 40)
           dah:
-            quantity_hint: *empty_input
+            quantity_hint: Number of daily attendance fee (41 to 50)
           daj:
-            quantity_hint: *empty_input
+            quantity_hint: Number of daily attendance fee (51+)
           dat:
-            quantity_hint: *empty_input
+            quantity_hint: Number of daily attendance fee (2+)
           saf:
-            quantity_hint: *empty_input
+            quantity_hint: Number of standard appearance fee
           pcm:
-            quantity_hint: *empty_input
+            quantity_hint: Number of plea and trial preparation hearing
           cav:
-            quantity_hint: *empty_input
+            quantity_hint: Number of conferences and views
```

Comment:
Have you tried this out. It may be pointless because the fee calculator JS will overwrite this with out from the api anyway. The "Number of days" hint is [set here I believe](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/261e13a847c6d99cdd30e8c37439ea38ffa92b8f/app/webpack/javascripts/modules/external_users/claims/fee_calculator/FeeCalculator.UnitPrice.js#L147)

<img width="253" alt="Screen Shot 2020-10-16 at 09 22 42" src="https://user-images.githubusercontent.com/7016425/96234335-3774ad00-0f91-11eb-9794-747db5f27ac2.png">

Reply:
This behaviour is also feature tested - [such as the here](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/features/fee_calculator/advocate/graduated_fee_calculator.feature#L50:L59)